tests/Baseline/spicy.types.unit.params-reference/output

```diff
@@ -1,4 +1,4 @@
 ### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
 X, old
-Y, new
-struct_, [$x="xxx"]
+Y, old
+struct_, [$x=""]
```

Comment:
This looks like an unexpected semantic change?

Reply:
This was due to the optimizer code which detects unreachable statements not handling unit parameters correctly (it only looked for `inout` parameters, but not parameter values which might be aliasing). I lifted `_isAliasingType` into `QualifiedType` and extended the parameter check.